python/pyspark/sql/connect/group.py

```diff
@@ -34,6 +34,7 @@
 from pyspark.util import PythonEvalType
 from pyspark.sql.group import GroupedData as PySparkGroupedData
 from pyspark.sql.pandas.group_ops import PandasCogroupedOps as PySparkPandasCogroupedOps
+from pyspark.sql.pandas.functions import _validate_pandas_udf  # type: ignore[attr-defined]
```

Comment:
Spark Classic invoke `pandas_udf` in Pandas Functions (ApplyInXXX), `pandas_udf` includes the function validation.
While in Spark Connect, we can not use `pandas_udf` due to the differences in underlying implementations: `pandas_udf` returns a wrapper while Spark Connect requires a `UserDefinedFunction` object.